frontend/components/home.jsx

```diff
@@ -22,51 +22,51 @@ const NumContainer = styled.div`
   color: #d6928b;
 `;
 
-const TableButton = styled(Button)`
+const Container = styled.div`
   height: 300px;
   width: 150px;
   font-size: 100px;
 `;
 
-const TensButton = styled(TableButton)`
+const TensContainer = styled(Container)`
   border-width: 1px 0 0 1px;
   margin: 5px 0 0 5px;
   border-radius: 25px 0 0 0;
   background-color: #f8dfca;
 `;
 
-const OnesButton = styled(TableButton)`
+const OnesContainer = styled(Container)`
   border-width: 1px 1px 0 0;
   margin: 5px 5px 0 0;
   border-radius: 0 25px 0 0;
   background-color: #bcd7cf;
 `;
 
-const EnterButton = styled(SubmitButton)`
-  height: 60px;
-  width: 300px;
-  border-radius: 0 0 25px 25px;
-  border-top: 0;
-  margin-top: 0;
-  background-color: #d6928b;
-`;
-
 const UpIcon = styled(FontAwesomeIcon).attrs({
   icon: 'chevron-up',
 })`
-  height: 25%;
-  font-size: 0.25em;
-  color: white;
-  padding: 0 2em;
+height: 25%;
```

Comment:
indent?

Reply:
huh didn't notice it shifted, good catch